Libraries/LibGfx/Font/Font.h

```diff
@@ -46,10 +47,27 @@ enum FontWidth {
     UltraExpanded = 9
 };
 
+enum class FontSlant : u8 {
+    Upright,
+    Italic,
+    Oblique
```

Comment:
For reference: oblique can also come with an angle parameter. So might be worth building in support for that.